### PR TITLE
RLS: 0.20.1

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: pandas-{{ version }}.tar.gz
   url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
-  sha256: 78bb2b39c8a1ac508f640ac489deda34ffecf9989d07bf2ee800d9aa449f78d7
+  sha256: e0613cb3847d5e0eecf4735e067835c34f41c2f4b6e8d2f5ef01acd28db74c39
 
 build:
   number: 1
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml]: This can actually be changed back to 0

Comment (TomAugspurger) [inline recipe/meta.yaml]: Ahh shoot, ok.

Is it OK to let the CI finish first, change it, and then merge that commit? Don't want to put more strain on the CI servers than we have to. Or is there any harm in leaving it at 1?

Comment (jorisvandenbossche) [inline recipe/meta.yaml]: Not sure. I don't think there is any harm in leaving it. But I also think it is ok to add a commit to just change that, but directly merge it (as the current commit shows that everything builds fine)

Comment (TomAugspurger) [inline recipe/meta.yaml]: OK. Once Appveyor finishes I'll push a commit resetting the build number to 0.

**Comment by TomAugspurger** [comment]
Everything finished green. Pushing the build reset commit now.

**Comment by dhirschfeld** [comment]
@TomAugspurger - It seems only the linux packages got built? Is it planned to release Windows packages?

https://anaconda.org/conda-forge/pandas/files

**Comment by ocefpaf** [comment]
@TomAugspurger there is a checksum mismatch,   GitHub does that some times... We need a PR to update that and get the Windows and OS X builds. 

I can do that tomorrow morning if you don't beat me to it.

**Pull request merged**